turbo.json

```diff
@@ -23,7 +23,8 @@
 		"highlight.io#build": {
 			"dependsOn": ["^build"],
 			"outputs": [".next/**"],
-			"env": ["GRAPHCMS_TOKEN"]
+			"env": ["GRAPHCMS_TOKEN"],
+			"cache": true
```

Comment:
confused why this helps since the `cache: false` is only for `dev`. i would expect the `frontend:dev` to not be cached, but depend on `highligt.io#build` which should already be cached? `highlight.io#dev` still isn't cached, but that shouldn't matter?

Reply:
I think it helps because `dev:frontend` runs `turbo run dev`, so it uses the config under `dev` in `turbo.json`. That's my understanding at least, but I could definitely be misunderstanding 😅